packages/block-library/src/query/edit/inspector-controls/per-page-control.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { RangeControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const MIN_POSTS_PER_PAGE = 1;
+const MAX_POSTS_PER_PAGE = 50;
```

Comment:
The current per-page option in the toolbar sets a max of 100 posts. So I think this should probably also be set to 100 to match.

https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/query/edit/query-toolbar.js#L53-L65

Reply:
Good call @justintadlock, I've update the numbers to match.